### DISCUSSION TITLE
Generate Tessellation with partial raster

### DISCUSSION THREAD

**Discussion by AmitS07** [neperfepx/neper#679] [Q&A] [answered]
As part of a study, I have far-field raster file information for a domain of size (1, 0.96, 1). This is the domain of study. To eliminate boundary effects, I would like to include a few grains to create buffer on both top and bottom of the domain of interest (shown in attached figure). Any help with the model building would be great. 
![Problem](https://user-images.githubusercontent.com/67289970/231220738-01d0d977-601c-41bf-a4dd-7c9701e74cfc.png)

**Comment by rquey** [comment, marked as answer]
I suppose that by "far-field" you mean grain centers and volumes, and you know the shape and size of the domain.

In the standard case, you would need to define the domain size using [`-domain`](https://neper.info/doc/neper_t.html?#cmdoption-domain) and specify the grain centers and volumes using [`-morpho centroiddiameq:file(<filename>)`](https://neper.info/doc/neper_t.html?highlight=centroiddiameq#cmdoption-morpho).

In your case, to add the buffers, you will need to define a slightly larger domain, and to add data to the centroid/diameq file
for the grains within the buffers.

This being said, and on a more scientific front, since the grains in the buffer zones won't be "real" grains, I don't think that considering them (versus imposing boundary conditions directly to the region of interest) would add any value to the simulation.
